Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -191,19 +197,23 @@ public static MixedRealityInteractionMapping[] GetMappingsForControllerType(Syst
             {
                 return WindowsMixedRealityController.DefaultInteractions;
             }
-            else if (controllerType == typeof(OculusTouchController))
```

Comment:
Why change this?

Reply:
cause we return in every other swtich. Pretty much redundant else.